### ISSUE TITLE
Angular 9 Version support 

### ISSUE THREAD

**Issue by Sijoma** [lephyrus/ngx-translate-messageformat-compiler#53]
Hey guys,

we use the latest version (4.5.0) however after updating to angular 9 we receive the following error:

`zone.js:699 Unhandled Promise rejection: Can't resolve all parameters for TranslateMessageFormatCompiler: (?). ; Zone: <root> ; Task: Promise.then ; Value: Error: Can't resolve all parameters for TranslateMessageFormatCompiler: (?).
    at getUndecoratedInjectableFactory (core.js:11397)
    at injectableDefOrInjectorDefFactory (core.js:11387)
    at providerToFactory (core.js:11451)
    at providerToRecord (core.js:11417)
 [...]`

Does someone have an idea about this problem?

**Comment by psoares-resilient** [comment]
Hey, apparently some injection decorator was missed somewhere here... Probably a missing `providedIn` ... Whilst they fix If, and it helps, you can extend the class  as is and decorate it yourself. Everything worked fine after that:

```
@Injectable({ providedIn: 'root' })
export class InjectableTranslateMessageFormatCompiler extends TranslateMessageFormatCompiler {}
```

and then use it in the options that are passed in the `.forRoot`. i.e.:

```
export const translateModuleConfig = {
  loader: {
    provide: TranslateLoader,
    useFactory: HttpLoaderFactory,
    deps: [HttpClient],
  },
  compiler: {
    provide: TranslateCompiler,
    useClass: InjectableTranslateMessageFormatCompiler,
  },
  isolate: false,
};
```

**Comment by marcobuschini** [comment]
@psoares-resilient why don't you fork the repository, make the change, and open a pull request? This will help them, I think.

**Comment by psoares-resilient** [comment]
> @psoares-resilient why don't you fork the repository, make the change, and open a pull request? This will help them, I think.

Hi, I might just do that! I need a couple hours to tidy up what I'm currently working on and if everything is hunky dori, I'll create a PR

**Comment by 7jpsan** [comment]
PR is open in https://github.com/lephyrus/ngx-translate-messageformat-compiler/pull/55

Make sure to upvote so it might get merged quickly :)

@Sijoma @marcobuschini 


p.s.: @psoares-resilient is just my work github account...

**Comment by lephyrus** [comment]
Fixed in v4.6.0.
